nas/handler.go

```diff
@@ -36,28 +36,30 @@ func HandleNAS(ue *context.RanUe, procedureCode int64, nasPdu []byte) {
 		} else {
 			if amfSelf.EnableSctpLb {
 				/* checking the guti-ue belongs to this amf instance */
```

Comment:
How is this (`checking the guti-ue belongs to this amf instance`) done when the db is disabled?

Reply:
When DB is not enabled it is not possible to have multiple AMFs, so there is no possibility that the UE belongs to a different AMF.